core/src/App.svelte

```diff
@@ -1703,6 +1575,134 @@
+  <Modal
+    settings="{mfDrawer.settings}"
+    nodepath="{mfDrawer.nodepath}"
+    on:close="{closeDrawer}"
+    on:drawerState="{setDrawerState}"
+    on:iframeCreated="{drawerIframeCreated}"
+    on:wcCreated="{drawerWCCreated}"
+  >
+    <Backdrop area="drawer" disable="{disableBackdrop}"/>
+  </Modal>
+  {/if} {#if confirmationModal.displayed}
+  <ConfirmationModal
+    settings="{confirmationModal.settings}"
+    on:modalConfirm="{()=> handleModalResult(true)}"
+    on:modalDismiss="{() => handleModalResult(false)}"
+  />
+  {/if} {#if internalUserSettingsObject.displayed}
+  <UserSettingsDialog
+    on:close="{closeUserSettings}"
+    userSettingGroups="{internalUserSettingsObject.userSettingGroups}"
+    bind:storedUserSettings
+  />
+  {/if}
+  <Backdrop disable="{disableBackdrop}">
+    <div
+      class="fd-page iframeContainer"
+      class:lui-split-view="{mfSplitView.displayed}"
+      class:lui-collapsed="{mfSplitView.collapsed}"
+      tabindex="0"
+      use:init
+    >
+      <Backdrop area="main" disable="{disableBackdrop}"/>
+      <div class="wcContainer"></div>
+    </div>
+    {#if mfSplitView.displayed}
+    <SplitView
+      splitViewSettings="{mfSplitView.settings}"
+      collapsed="{mfSplitView.collapsed}"
+      nodepath="{mfSplitView.nodepath}"
+      on:iframeCreated="{splitViewIframeCreated}"
+      on:statusChanged="{splitViewStatusChanged}"
+      on:wcCreated="{splitViewWCCreated}"
+      disableBackdrop="{disableBackdrop}"
+    />
+    {/if}
+  </Backdrop>
+  {#if showLoadingIndicator}
+  <div
+    in:fade="{{delay: 250, duration: 250}}"
+    out:fade="{{duration: 250}}"
+    class="fd-page spinnerContainer"
+    aria-hidden="false"
+    aria-label="Loading"
+  >
+    <div
+      class="fd-busy-indicator--m"
+      aria-hidden="false"
+      aria-label="Loading"
+      data-testid="luigi-loading-spinner"
+    >
+      <div class="fd-busy-indicator--circle-0"></div>
+      <div class="fd-busy-indicator--circle-1"></div>
+      <div class="fd-busy-indicator--circle-2"></div>
+    </div>
+  </div>
+  {/if}
+  <TopNav
+    pathData="{navigationPath}"
+    pathParams="{pathParams}"
+    on:handleClick="{handleNavClick}"
+    on:resizeTabNav="{onResizeTabNav}"
+    on:toggleSearch="{toggleSearch}"
+    on:closeSearchResult="{closeSearchResult}"
+    on:handleSearchNavigation="{handleSearchNavigation}"
+    bind:isSearchFieldVisible
+    bind:displaySearchResult
+    bind:displayCustomSearchResult
+    bind:searchResult
+    bind:inputElem
+    bind:luigiCustomSearchRenderer__slot
+    burgerTooltip="{burgerTooltip}"
+  />
+  {#if !(hideNav)}
+  <GlobalNav
+    pathData="{navigationPath}"
+    pathParams="{pathParams}"
+    on:handleClick="{handleNavClick}"
+  />
+  {#if breadcrumbsEnabled}
+  <Breadcrumb
+    pathData="{navigationPath}"
+    pathParams="{pathParams}"
+    on:handleClick="{handleNavClick}"
+  />
+  {/if} {/if} {#if !(hideNav||hideSideNav)}
```

Comment:
@rafalgamon, the prettier setting seems not applied.
Did you follow this document? https://wiki.wdf.sap.corp/wiki/display/luigi/Setup+VS+Code+auto+formatting+for+.svelte+files